### PR TITLE
Add Spanish translation of pr_checks.mdx

### PR DESCRIPTION
**Pull request by donelianc**
# What does this PR do?

Add the Spanish translation for `pr_checks.mdx` as part of the #15947 issue.
Changes include the Spanish version of the original document and the updated `_toctree.yml` file.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests) Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? _Task assignment [here](https://github.com/huggingface/transformers/issues/15947#issuecomment-1321245149)_.
- [x] Did you make sure to update the documentation with your changes? Here are the [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

### CODE REVIEW

**Comment by donelianc** [comment]
Hi @osanseviero. Can you help review this PR, please?

File: docs/source/es/pr_checks.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+El proceso `build_pr_documentation` compila y genera una vista previa de la documentación para asegurarse de que todo se vea bien una vez que se fusione tu PR. Un bot agregará un enlace para obtener una vista previa de la documentación en tu PR. Cualquier cambio que realices en el PR se actualiza automáticamente en la vista previa. Si la documentación no se genera, haz clic en **Detalles** junto al proceso fallido para ver dónde salió mal. A menudo, el error es tan simple como que falta un archivo en `toctree`.
+
+Si estás interesado en compilar u obtener una vista previa de la documentación localmente, echa un vistazo al [`README.md`](https://github.com/huggingface/transformers/tree/main/docs) en la carpeta `docs`.
+
+## Estilo de código y documentación.
+
+El formato de código se aplica a todos los archivos fuente, los ejemplos y las pruebas utilizando `black` e `isort`. También tenemos una herramienta personalizada que se ocupa del formato de los archivos _docstrings_ y `rst` (`utils/style_doc.py`), así como del orden de las importaciones _lazy_ realizadas en los archivos `__init__.py` de Transformers (`utils /custom_init_isort.py`). Todo esto se puede probar ejecutando
```
Comment (osanseviero) [inline docs/source/es/pr_checks.mdx:74]:
```suggestion
El formato de código se aplica a todos los archivos fuente, los ejemplos y las pruebas utilizando `black` e `isort`. También tenemos una herramienta personalizada que se ocupa del formato de los _docstrings_ y archivos `rst` (`utils/style_doc.py`), así como del orden de las importaciones _lazy_ realizadas en los archivos `__init__.py` de Transformers (`utils /custom_init_isort.py`). Todo esto se puede probar ejecutando
```

File: docs/source/es/pr_checks.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+make repo-consistency
+```
+
+Esta instrucción verifica que:
+
+- Todos los objetos agregados al _init_ están documentados (realizados por `utils/check_repo.py`)
+- Todos los archivos `__init__.py` tienen el mismo contenido en sus dos secciones (realizado por `utils/check_inits.py`)
+- Todo el código identificado como una copia de otro módulo es consistente con el original (realizado por `utils/check_copies.py`)
+- Todas las clases de configuración tienen al menos _checkpoint_ válido mencionado en sus _docstrings_ (realizado por `utils/check_config_docstrings.py`)
+- Las traducciones de los README y el índice del documento tienen la misma lista modelo que el README principal (realizado por `utils/check_copies.py`)
```
Comment (osanseviero) [inline docs/source/es/pr_checks.mdx:108]:
```suggestion
- Las traducciones de los README y el índice del documento tienen la misma lista de modelos que el README principal (realizado por `utils/check_copies.py`)
```

File: docs/source/es/pr_checks.mdx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+- Las tablas generadas automáticamente en la documentación están actualizadas (realizadas por `utils/check_table.py`)
```
Comment (osanseviero) [inline docs/source/es/pr_checks.mdx:109]:
```suggestion
- Las tablas generadas automaticamente en la documentación están actualizadas (realizadas por `utils/check_table.py`)
```

**Review by osanseviero — APPROVED** [review]
Thanks a lot! 🔥 I left some minor comments but overall looks amazing!

**Comment by donelianc** [comment]
Only minor typos were spotted during the first review. I already addressed them with my latest commit. 
@sgugger, if you agree, we can skip a second review from [osanseviero](https://github.com/osanseviero) and merge this PR. 

Thanks!

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**